.cicdtemplate/.github/workflows/deploy_staging_and_production_to_firebase_app_distribution.yml

```diff
@@ -46,16 +46,15 @@ jobs:
           name: DetektReports
           path: build/reports/detekt/
 
-      - name: Run unit tests and Jacoco
-        run: ./gradlew jacocoTestReport
+      - name: Run unit tests with Kover
+        run: ./gradlew koverMergedHtmlReport
 
       - name: Archive code coverage reports
         uses: actions/upload-artifact@v2
         with:
           name: CodeCoverageReports
           path: |
-            app/build/reports/jacoco/jacocoTestReport/
-            data/build/reports/jacoco/jacocoTestReport/
+            build/reports/kover/merged/
```

Comment:
Could be minimised to `path: build/reports/kover/merged/`

Same goes for `run_detekt_and_unit_tests.yml` 🚀